en/dev_setup/dev_env_linux_centos.md

```diff
@@ -30,9 +30,62 @@ You may want to also install `python-pip` and `screen`.
 :::
 
 ## GCC Toolchain Installation
-<!-- import GCC toolchain common documentation -->
-{% include "_gcc_toolchain_installation.md" %}
+<!-- GCC toolchain documentation used for all Linux platforms to build NuttX -->
 
+Execute the script below to install GCC 7-2017-q4:
```

Comment:
The version is quite outdated although it probably still works without issues except maybe fmu-v2 flash size because the old compiler has less optimizations. I just never use CentOS anymore and would not be sure if it's worth testing on there.

Reply:
Thanks. I will fix this as part of https://github.com/PX4/PX4-user_guide/pull/1064 (by marking this stuff as untrusted :-) )